src/App.tsx

```diff
@@ -1,13 +1,56 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+// import { UserWarning } from './UserWarning';
+// if (!USER_ID) {
+//   return <UserWarning />;
+// }
 
-const USER_ID = 0;
+import React, { useState, useEffect } from 'react';
+import cn from 'classnames';
+import { TodoList } from './components/TodoList';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Error } from './types/Error';
+import { Filter } from './types/Filter';
+
+const USER_ID = 10267;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<Error | null>(null);
+  const [filter, setFilter] = useState<Filter>(Filter.ALL);
+
+  const addError = (error: Error) => {
+    setErrorMessage(error);
```

Comment:
addError function can be wrapped with useCallback to prevent unnecessary re-renders.